src/index.js

```diff
@@ -14,21 +14,124 @@ function createElement(tagName, props, ...children) {
       element.appendChild(child);
       return;
     }
+
     element.appendChild(document.createTextNode(child));
   });
 
   return element;
 }
 
-function render() {
+function render({
+  operand1 = 0, operand2 = 0, operator = '', result = 0,
+}) {
+  const calculatorElements = {
+    operand1,
+    operand2,
+    operator,
+    result,
+  };
+
+  function calculator() {
```

Comment:
`calculator` 함수가 인자로 `operand1, operand2, operator ` 를 받게되면 calculator 함수도 pure function 이라고 할수 있겠죠?? (만약 이대로 바꾸시게된다면, 이름은 중복되지 않게 바꾸시거나, render 함수 밖으로 빼는걸 추천드립니다 !)
```suggestion
  function calculator({ operand1, operand2, operator }) {
```